wordpress-mobile/WordPress-iOS
# Update BloggingPrompt Transformer class to NSSecureUnarchiveFromData

Refs #18762

This gets rid of the warning that you see in the console every time the app parses `BloggingPrompt`:

```
'NSKeyedUnarchiveFromData' should not be used for un-archiving and will be removed in a future release.
```

---

⚠️ **Auto-merge is turned on.** ⚠️ 

---

## To test

- Go to Prompt List.
- Verify that the warnings no longer show up.

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually compiled and tested to ensure that it runs fine.

3. What automated tests I added (or what prevented me from doing so)
N/A. Core Data model changes.

PR submission checklist:
